lib/WorkflowMain.groovy

```diff
@@ -0,0 +1,93 @@
+//
+// This file holds several functions specific to the main.nf workflow in the nf-core/rnaseq pipeline
+//
+
+class WorkflowMain {
+
+    //
+    // Citation string for pipeline
+    //
+    public static String citation(workflow) {
+        return "If you use ${workflow.manifest.name} for your analysis please cite:\n\n" +
+            "* The pipeline\n" +
+            "  https://doi.org/10.5281/zenodo.1400710\n\n" +
```

Comment:
We shouldn't have a zenodo doi for this pipeline yet

Reply:
Yes, I just commented it and mention as "unpublished"